src/view/components/interface.ts

```diff
@@ -0,0 +1,117 @@
+import type { BaseItemDto } from '@jellyfin/sdk/lib/generated-client';
```

Comment:
Should this be a type definition file? i.e. `interface.d.ts`

Reply:
no need here `d.ts` files are treated as an ambient (global) module declarations only if they don't have any imports.